src/coreclr/src/debug/daccess/request.cpp

```diff
@@ -4500,3 +4500,29 @@ HRESULT ClrDataAccess::GetMethodsWithProfilerModifiedIL(CLRDATA_ADDRESS mod, CLR
 
     return hr;
 }
+
+HRESULT ClrDataAccess::GetAssemblyLoadContext(CLRDATA_ADDRESS methodTable, CLRDATA_ADDRESS* assemblyLoadContext)
+{
+    if (methodTable == 0 || assemblyLoadContext == NULL)
+        return E_INVALIDARG;
+
+    SOSDacEnter();
+    PTR_MethodTable pMT = PTR_MethodTable(TO_TADDR(methodTable));
```

Comment:
>TO_TADDR [](start = 42, length = 8)

Can you please change this to CLRDATA_ADDRESS_TO_TADDR

Reply:
Sure. Btw, I am redoing it based on @jkotas feedback to simplify things considerably, so please wait with reviewing a bit.